virttest/qemu_vm.py

```diff
@@ -1459,7 +1470,8 @@ def add_disable_modern(devices, dev, dev_type):
                     if params.get('machine_type').startswith("arm64-mmio"):
                         dev = QDevice('virtio-serial-device')
                     else:
-                        dev = QDevice('virtio-serial-pci', parent_bus=pci_bus)
+                        parent_bus = _get_pci_bus(port_params, "virtio_port")
```

Comment:
I think connect virtio pci device to downstream port or pcie.0 it acceptable, in my mind, if virtio pci device connect to pcie.0 hotplug/unplug is not allowed (I'm not sure about this).
Device tree model in Q35 is very very flexible,  framkly speaking it very hard to cover all in this time.